anitya/lib/backends/__init__.py

```diff
@@ -199,33 +199,28 @@ def call_url(self, url, insecure=False):
 
         :arg url: the url to request (get).
         :type url: str
+        :arg insecure: flag for secure/insecure connection (default False)
+        :type insecure: bool
+        :type use_token: bool
         :return: the request object corresponding to the request made
         :return type: Request
         '''
-        user_agent = 'Anitya %s at upstream-monitoring.org' % \
-            pkg_resources.get_distribution('anitya').version
-        from_email = anitya_config.get('ADMIN_EMAIL')
-
+        headers = self.get_header()
         if '*' in url:
             url = self.expand_subdirs(url)
 
         if url.startswith('ftp://') or url.startswith('ftps://'):
             socket.setdefaulttimeout(30)
 
             req = urllib2.Request(url)
-            req.add_header('User-Agent', user_agent)
-            req.add_header('From', from_email)
+            req.add_header('User-Agent', headers.user_agent)
+            req.add_header('From', headers.from_email)
```

Comment:
``headers`` is a dictionary, right? I don't think this works.

Reply:
Yes, you are right. This means, that there was not test for this, I will add it.